src/fragments/lib/auth/ios/mfa_preference/20_update_mfa_preference.mdx

```diff
@@ -0,0 +1,48 @@
+<BlockSwitcher>
+
+<Block name="Async/Await">
+
+```swift
+func updateMFAPreferences() async throws {
+    let authCognitoPlugin = try Amplify.Auth.getPlugin(
+        for: "awsCognitoAuthPlugin") as? AWSCognitoAuthPlugin
+
+    let smsMfaPreference: MFAPreference = .preferred
```

Comment:
the code snippet seems odd to me that it would include both SMS and TOTP for each of the SMS and TOTP section individually. Can we update that? @dnys1